src/platform/mock-local-home-platform.ts

```diff
@@ -46,9 +46,9 @@ export class MockLocalHomePlatform implements MockUDPListener {
   private static instance: MockLocalHomePlatform;
 
   private udpScanConfigs: UDPScanConfig[] = [];
-  private mockNetwork: MockNetwork;
-  private deviceManager: smarthome.DeviceManager;
-  private app: AppStub;
+  private mockNetwork!: MockNetwork;
```

Comment:
Doesn't the `!` operator assert that the variable is not null? Does that make sense in this context?

Reply:
`tslint` was complaining these value were never initialized and may be undefined, but `mockNetwork` and `app` are set in setters.  I can see how this isn't the best solution, so I just replaced `!` with ` | undefined` types, and null checks for `app` and `mockNetwork` when they are actually used.  For `deviceManager` I added a default value.